eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -26,6 +26,76 @@ steps:
     inlineScript: |
       Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
+- task: CredScan@2
+  continueOnError: "false"
```

Comment:
`false` is the default for continueOnError, you don't need to explicitly set it here, or on line 36 below.

Reply:
Removed.